source/features/deep-blame.tsx

```diff
@@ -0,0 +1,111 @@
+import './deep-blame.css';
+import select from 'select-dom';
+import * as api from '../libs/api';
+import React from 'dom-chef';
+import features from '../libs/features';
+import {getRepoGQL, getReference} from '../libs/utils';
+import delegate, {DelegateEvent} from 'delegate-it';
+import versionIcon from 'octicon/versions.svg';
+import octofaceIcon from 'octicon/octoface.svg';
```

Comment:
Sort these so they look like a tree 😃 refer to the look in my other features. This makes it slightly easier to follow because they’re closer to being “aligned”

And always put the npm modules above the local `./` and `../` files

Reply:
I knew you were going to comment on that LOL

Reply:
Why did you let me comment then 🤷‍♂️

Reply:
I forgot to change it. I was saying when I was writing the code....